messaging/src/ios/messaging.mm

```diff
@@ -180,7 +180,6 @@ void NotifyListenerSet(Listener* listener) {
   FIREBASE_ASSERT_RETURN(RequestPermissionLastResult(), internal::IsInitialized());
 
   if (RequestPermissionLastResult().status() == kFutureStatusPending) {
-    LogError("Status is pending. Return the pending future.");
```

Comment:
Could be a LogDebug if we still need this for some reason.

Reply:
Ah, I merged before seeing this comment.  Do you think I should add `LogDebug` back?

I did a code search and it does not seems like we are logging anything when LastResult is pending.